helm/yatai-image-builder/values.yaml

```diff
@@ -91,17 +84,6 @@ dockerRegistry:
   useAWSECRWithIAMRole: false
   awsECRRegion: ap-northeast-3
 
-aws:
```

Comment:
I think we still need it?

Reply:
I don’t think we set these values right now, do we?

Reply:
can not delete these as well. it's used in bentocloudctl since bentocloudctl can not use pre-signed url.